stdlib/string.lua

```diff
@@ -52,3 +52,41 @@ function string.split(s, sep, pattern)
     s:gsub(pattern, function(c) fields[#fields+1] = c end)
     return fields
 end
+
+--- Splits a string by another string
+-- @param s the string to split
+-- @param sep the boundary string
+-- @param limit If limit is set and positive, the returned array will contain a maximum of limit elements with the last element containing the rest of string. If the limit parameter is negative, all components except the last -limit are returned. If the limit parameter is zero, then this is treated as 1.
+-- @return table
+function string.explode(s, sep, limit)
+    if not sep or sep == "" then return false end
```

Comment:
No worries, I just thought the `return false` would be confusing for a method that reports it returns a table.